jsamchineme/send-it
# Ft change password frontend 162825851

#### What does this PR do?
- Implement password change functionality
#### Description of Task to be completed?
- Add password reset request functionality
- Add frontend route to handle click from reset link sent via email
- Add password change functionality
#### How should this be manually tested?
- pull branch and run _npm run start_
- navigate to http://localhost:8001/forgot-password
- request password reset providing your email (meaning you should have an account already)
- an email will be sent to you on successful request, click on the link in the mail and you will get redirect to the site
- the backend set the base url to the production url, so on load of the route, copy the link and change the base uri to http://localhost:8001, so that you now have a link like this
`http://localhost:8001/password-reset/email/{youremail}/token/{resetToken}`
- the route loads the change password page, where you can now provide the new password
- provide new password and now try logging in
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162825851
#### Screenshots (if appropriate)
None
#### Questions:`
None
